src/assets/css/header.scss

```
@@ -0,0 +1,192 @@
+.mor-links-container {
+    background-color: $header-background;
+    box-sizing: border-box;
+    padding: 60px 75px;
+    position: relative;
+    width: 320px;
+    transition-duration: 500ms;
+    top: 0;
+  
+    img {
+      height: auto;
+      margin-bottom: 100px;
+      max-width: 100%;
+    }
+  
+    nav {
+      ul {
+        margin:0;
+        margin-bottom: 100px;
+        padding-inline-start: 0;
+      }
+    }
+  
+    .social {
+      justify-content: space-between;
+      display: flex;
+      margin-bottom: 75px;
+  
+      a {
+        color: $nav-link-color;
+        display: inline-block;
+        font-weight: 400;
+        transition: all 500ms ease 0s;
+        text-decoration: none;
+        outline: 0 solid transparent;
+      
+        &:hover {
+          color: $line-bgcolor;
+          transition: all 500ms ease 0s;
+        }
+      } 
+    }
+  
+    @media only screen and (max-width: 1023px) and (min-width: 768px){
+      width: 280px;
+      padding: 50px 50px;
+      left: 0;
+      transition-duration: 500ms;
+    }
+  
+    @media screen and (max-width: 767px) and (min-width: 320px){
```

same here, you can do:
```suggestion
    @media screen and (max-width: #{$breakpoint-tablet - 1}) and (min-width: $breakpoint-mobile){
```